doc/JSON-RPC-interface.md

```diff
@@ -36,3 +36,113 @@ However, the wallet may not be up-to-date with the current state of the mempool
 or the state of the mempool by an RPC that returned before this RPC. For
 example, a wallet transaction that was BIP-125-replaced in the mempool prior to
 this RPC may not yet be reflected as such in this RPC response.
+
+## Basic usage
+
+Launch the daemin in the background:
```

Comment:
Nit / later: explain that as an alternative you can run the daemon in one tab and use bitcoin-cli in another. That way you can keep an eye on logs. Or just mention `tail -f <datadir>/<testnet3 or regtest>/debug.log`

See also #15438.